src/Spain/Mews.Fiscalizations.Spain/Communication/SoapClient.cs

```diff
@@ -62,7 +62,7 @@ private static ResponseResult<TOut> DeserializeSoapError<TOut>(string response)
             }
             catch (Exception e)
             {
-                throw new InvalidOperationException($"Fail to parse response {response}", e);
+                throw new InvalidOperationException($"Failed to parse response.", e);
```

Comment:
Hold on, why did we drop the response? That was the only justification of this whole exception catching. If we don't expose the response, the whole try catch is useless.